bramkouwen/Rock-Paper-Scissors-Game
# Writing quit functionality

## What does this PR do?
* It updates the game.py file to contain a quit function

## How do I check this PR?
* Pull down the branch 
* See the quit function added to the game.py file at line 51, 56-58

## Screenshots 
None.
